packages/examples/packages/send-flow/src/index.tsx

```diff
@@ -98,7 +100,29 @@ export const onUserInput: OnUserInputHandler = async ({
   if (event.type === UserInputEventType.InputChangeEvent) {
     switch (event.name) {
       case 'amount':
-      case 'to':
+      case 'to': {
+        // For testing purposes, we display the avatar if the address is a valid hex checksum address.
+        if (is(event.value, HexChecksumAddressStruct)) {
```

Comment:
This will only enable it and never disable it right? Can we just update the `displayAvatar` prop whenever it changes?

Reply:
Yeah, changed.